src/components/events/EventsSection.jsx

```diff
@@ -18,23 +19,29 @@ class EventsSection extends React.Component {
   }
 
   componentDidMount() {
-    db.collection('events').onSnapshot((querySnapshot) => {
-      const events = [];
-      querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, eventDate } = doc.data();
+    const events=[];
+    axios.get('http://localhost:5000/events').then((response) => {
```

Comment:
use the utils in `ApiUtils.js`

Reply:
Fixed in https://github.com/uwblueprint/dancefest-web/pull/94/commits/a82a0f7c81b6e069548c28f7d6a25becc52e2a1f, thanks!